convbench/conv_utils.py

```diff
@@ -33,30 +35,36 @@ class ConvConfig:
     Q: int
     F: int
     S: int
+    is_grouped_conv: bool
+    G: int # group count
+    is_3D_conv: bool
```

Comment:
possible to make another class instead for conv3d? That way every time we try to add a problem, we don't need the `False, -1, -1, -1` part.